apps/desktop/desktop/src/main/java/bisq/desktop/components/controls/validator/NumberValidator.java

```diff
@@ -152,6 +115,10 @@ protected void eval() {
         }
     }
 
+    private String cleanUpFormatting(String inputText) {
+        return inputText.replace(",", "");
+    }
+
     private boolean isNumber(String inputText) {
         return inputText.matches("^-?\\d+(\\.\\d+)?$");
```

Comment:
Does that respect any decimal and thousands separator format diff. locals?

Reply:
It does respect any decimals, but not the thousands separator (that's why the clean-up first). But we could have it implicitly if we define the locale.

I still see that inputs like this are accepted (although later we ignore the decimals):

![image](https://github.com/bisq-network/bisq2/assets/145597137/0533ce3c-5ff0-42e7-b101-ab0588b9e21f)

Therefore I propose just using `NumberFormat` to validate for now and if we decide on a locale, then we can make it more specific and target those corner cases better. (In another PR)